tests/test_runserver_cleanup.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+from aiohttp import ClientSession, ClientTimeout
+from pytest_toolbox import mktree
+import pytest
+from aiohttp_devtools.runserver import runserver
+from .conftest import forked
+from tempfile import NamedTemporaryFile
+
+
+# Note this code is heavily based on test_start_runserver from test_runserver_main.py
+# TODO: Can't find a way to fix these warnings, maybe fixed in aiohttp 4.
+@pytest.mark.filterwarnings(r"ignore:unclosed:ResourceWarning")
+@forked  # forked doesn't run on Windows and is skipped - see cleanup_app.py instead
+def test_server_cleanup(tmpworkdir, smart_caplog):
```

Comment:
I'm not sure what this test is testing?
Seems like it verifies that cleanup is run, but the test itself is calling cleanup. i.e. It doesn't test any behaviour of aiohttp-devtools, only aiohttp and the test code itself.

Reply:
That's a good point and I've replaced this test in 21cd7a2fdc5d41abdf94a4e768f77034f0ccbf7a.